evaluate.py

```diff
@@ -132,23 +128,38 @@ def run_test(init_op, dataset):
 
             bar.finish()
 
+            # Print test summary
             wer, cer, samples = calculate_report(wav_filenames, ground_truths, predictions, losses)
             mean_loss = np.mean(losses)
+            print('Test on %s - WER: %f, CER: %f, loss: %f' % (dataset, wer, cer, mean_loss))
+            print('-' * 80)
 
-            # Take only the first report_count items
-            report_samples = itertools.islice(samples, FLAGS.report_count)
+            # Take only the first, median and last report_count items
+            best_samples = samples[:FLAGS.report_count]
+            worst_samples = samples[-FLAGS.report_count:]
+            median_index = int(len(samples) / 1.5)
```

Comment:
That seems like a weird definition of the median ?

Reply:
Your right^^
I did test something else with that...